### ISSUE TITLE
Voodoo Doll Crash

### ISSUE THREAD

**Issue by zBoringDrop** [konsolas/AAC-Issues#1495]
Hi, if i use voodoodoll plugin, and i hit the doll the server crash.
This is the error + thread: https://www.spigotmc.org/threads/voodoodoll.197467/#post-3134602

How i can fix it? Maybe with a bypass check? Or other?

**Comment by Photon-GitHub** [comment]
The hastebin is empty.

**Comment by zBoringDrop** [comment]
Try this: https://pastebin.com/X6Ha8J9U

**Comment by Janmm14** [comment]
@Photon-GitHub no, its not empty its just hastebin having problems again and again

**Comment by zBoringDrop** [comment]
So? Sufficient info?

**Comment by zBoringDrop** [comment]
News about this?

**Comment by zBoringDrop** [comment]
Ehm.. why inactivity? I was waiting you..
Unfortunatly i cant test, can giù do that? You must only download the plugin and hit the doll
